docs/usage-guide.md

```diff
@@ -0,0 +1,85 @@
+# Usage Guide for Osmo-Notify
+
+## Introduction
+
+Welcome to the usage guide for Osmo-Notify, a powerful notification management system designed to simplify the process of sending notifications through various channels. This guide will walk you through the steps of integrating Osmo-Notify into your application and making the most of its features.
+
+## Table of Contents
+
+1. Overview
+2. Pushing Data to the Database
+3. Using the Osmo-Notify API
+4. Tracking Notification Status
+5. Available Channel Types
+6. Delivery Status Information
+
+## 1. Overview
+
+Osmo-Notify offers a streamlined solution for sending notifications via different channels. It operates by monitoring the `notify_notifications` database table and automatically dispatches notifications for records with a `PENDING` status.
+
+## 2. Pushing Data to the Database
+
+Developers have the flexibility to populate the `notify_notifications` database table directly or opt for the recommended approach of utilizing the Osmo-Notify API. By following this method, the notification processing is seamlessly managed by Osmo-Notify.
+
+## 3. Using the Osmo-Notify API
+
+To use the Osmo-Notify API, follow these steps:
+
+- **Method:** POST
+- **Endpoint:** `/notifications`
+
+**Sample Request Body:**
+```json
+{
+  "channelType": 1,
+  "data": {
+    "from": "sender@example.com",
+    "to": "recipient@example.com",
+    "subject": "Test subject",
+    "text": "This is a test notification",
+    "html": "<b>This is a test notification</b>"
+  }
+}
+```
+
+**Response:**
+```json
+{
+  "notification": {
+    "channelType": "smtp-server",
+    "data": {
+      "from": "sender@example.com",
+      "to": "recipient@example.com",
+      "subject": "Test subject",
+      "text": "This is a test notification",
+      "html": "<b>This is a test notification</b>"
+    },
+    "deliveryStatus": 1,
+    "createdBy": "osmo-notify",
+    "createdOn": "2023-08-25T10:55:36.794Z",
+    "result": null,
+    "modifiedBy": null,
+    "modifiedOn": null,
+    "id": 1
+  }
+}
+```
+
+## 4. Tracking Notification Status
+
+Osmo-Notify updates the `deliveryStatus` and `result` columns to provide information on the notification's status. Use these columns to track the progress of your notifications.
```

Comment:
if you refer to column then better use the exact column name is it not `delivery_status`?

Reply:
Update column name